### PR TITLE
[patch] improved function testLibVersion

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

File: updater.js [hunk 1]
```diff
@@ -160,3 +162,4 @@ function testLibVersion(lib, version) {
     });
   });
 }
+
```
Comment (PeterDaveHello) [inline updater.js]: Nothing to do with this line?

Comment (LeaYeh) [inline updater.js]: i think it is better that add one more empty line in the end of program, didn't it @@?

Comment (PeterDaveHello) [inline updater.js]: What is the relation with the newline and your commit message?

**Comment by LeaYeh** [comment]
OK

**Pull request merged**